lib/board.rb

```diff
@@ -0,0 +1,83 @@
+require_relative './game.rb'
+
+class Board < Game
+  def initialize
+    @game = Game.new(@players)
+  end
+
+  def welcome_message
+    puts 'Welcome to Tic Tac Toe ! ! !'
+  end
+
+  def user_setup
+    print 'Player 1: What is your name?:'
+    @player_x = gets.chomp
+    @player_x.upcase!
+    name = true
+    while name
+      print 'Player 2: What is your name?:'
+      @player_y = gets.chomp
+      @player_y.upcase!
+      if @player_y != @player_x
+        name = false
+      else
+        puts 'Please Choose Different Name'
+      end
+    end
+  end
+
+  def start_game
+    puts "X is #{@player_x} and O #{@player_y}"
+    puts 'Game Start!'
+  end
```

Comment:
- [ ] No `puts/print` and `get.chomp` outside `main.rb` file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

- This method should be refactored and use a String variable to concatenate the messages and call it in your main.rb file like this: `puts start_game` or its content should be placed directly in the main.rb file